tables/rw-rw-g1.utb

```diff
@@ -0,0 +1,135 @@
+# liblouis: Kinyarwanda Grade 1 Braille
+#
+#-name: Kinyarwanda Grade 1 Braille
+#-index-name: Kinyarwanda Grade 1 Braille
+#-display-name: Kinyarwanda Grade 1 Braille
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#
+#+language:rw
+#+region:rw
+#+type:literary
+#+contraction:no
+#+grade:1
+#+direction:both
+#
+# Developed in collaboration with Rwanda Union of the Blind, Kigali, Rwanda
+# Copyright (C) 2022 Compass Braille, UK, https://.compassbraille.org
```

Comment:
I'm missing references to documentation of the braille code. Does it exist?

Reply:
I will try to find out but don't envisage this before 5th Sept release date.

Reply:
Done